scripts/export_comments.py

```diff
@@ -61,9 +62,17 @@ def create_comments_excel_file(survey_id, period, submissions):
 
 def get_all_submissions(survey_id, period):
     """Get all submissions that match the survey_id and period supplied"""
+    if survey_id == '181':
+        vacancy_records = []
+        for survey_id in ['182', '183', '184', '185']:
+            survey_records = session.query(SurveyResponse).filter(SurveyResponse.data['survey_id'].astext == survey_id)
+            records = survey_records.filter(SurveyResponse.data['collection']['period'].astext == period).all()
```

Comment:
Just 1 small change.  I think this should be `period_specific_records` as it's difficult at a glance to see what the difference is between `survey_records`, `records` and `vacancy_records`

Reply:
I meant `records` becomes `period_specific_records`..